plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -943,6 +962,11 @@ public boolean configure(final String name, final Map<String, Object> params) th
             _mountPoint = "/mnt";
         }
 
+        _libvirtConnectionProtocol = (String) params.get("libvirt.connection.protocol");
+        if (_libvirtConnectionProtocol == null) {
+            _libvirtConnectionProtocol = "qemu://";
```

Comment:
Isn't this redundant since you already specify qemu:// at the beginning of LibvirtComputingResource?